src/frontends/common/src/frontend.cpp

```diff
@@ -74,6 +75,7 @@ void FrontEnd::normalize(const std::shared_ptr<Model>& model) const {
 
 void FrontEnd::add_extension(const std::shared_ptr<ov::Extension>& extension) {
     if (m_actual) {
+        add_extension_to_shared_data(m_shared_object, extension);
```

Comment:
Can we have C++ test to verify the fix?

Reply:
Unfortunetelly, I'm not able to reproduce the problem on C++ level

Reply:
Why do we need this?
If I remember right we also have some so holders in some frontends:
https://github.com/openvinotoolkit/openvino/blob/ec3283ebe1dd4ee3b76f21fb9994d1da8b077154/src/frontends/ir/src/frontend.cpp#L107

Could we use the same approach? Or migrate to common approach.

Reply:
right, I think that shared_ptr holder for extensions in IR frontend is no more needed. I've removed it